db/migrations/20190603103434_drop_title_from_movies.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+exports.up = (Knex) => {
+  return Knex.schema.table('movies', (table) => {
+    table.dropColumn('title');
+  })
+  .then(() => {
+    return Knex.raw('ALTER TABLE movies ADD CONSTRAINT movies_name_not_null CHECK (name IS NOT NULL) NOT VALID')
+    .then(() => {
+      return Knex.raw('ALTER TABLE movies VALIDATE CONSTRAINT movies_name_not_null');
```

Comment:
Is this recommended from Myagi? I wonder what the locking behavior for `VALIDATE CONSTRAINT` is. (Not super relevant for this review, just thinking we should be careful before doing something like this in lob-api where tables are big.)